src/main/java/io/odpf/firehose/sink/SinkFactory.java

```diff
@@ -62,6 +65,11 @@ public void init() {
             case INFLUXDB:
             case ELASTICSEARCH:
             case REDIS:
+                redisSinkFactory = new RedisSinkFactory(
```

Comment:
adding it here is not correct.
for example if http sink is configured, it will create redissinkfactory too..

Reply:
you have to move REDIS: just before Log.